docs/treat-migration-guide.md

```diff
@@ -0,0 +1,304 @@
+# 🍬 treat migration guide
+
+🎉 First of all, thanks for using treat, and thanks for your interest in vanilla-extract!
+
+When we first started work on vanilla-extract, it actually began life as `treat-next` — an experiment with replacing our custom webpack-based theming system with CSS Variables. However, we quickly realised that the theming system was too different to carry the same name. That said, they're similar enough that migrating shouldn't be too hard (relative to migrating to another library).
+
+We've made sure that both treat and vanilla-extract can run simultaneously in the same project, so if you have a lot of treat files, don't feel like you have to migrate everything at once!
+
+## New file extension
+
+The file extension has changed from `*.treat.ts` to `*.css.ts`.
```

Comment:
Should we say default file extension? Also, mention `.js` as well or is that implied?

Reply:
I feel like it's implied—happy to leave it?